torch/csrc/Device.cpp

```diff
@@ -181,39 +181,39 @@ PyTypeObject THPDeviceType = {
   "torch.device",                        /* tp_name */
   sizeof(THPDevice),                     /* tp_basicsize */
   0,                                     /* tp_itemsize */
-  nullptr,                                     /* tp_dealloc */
-  nullptr,                                     /* tp_print */
-  nullptr,                                     /* tp_getattr */
-  nullptr,                                     /* tp_setattr */
-  nullptr,                                     /* tp_reserved */
+  0,                                     /* tp_dealloc */
+  0,                                     /* tp_print */
+  0,                                     /* tp_getattr */
+  0,                                     /* tp_setattr */
+  0,                                     /* tp_reserved */
```

Comment:
We only need to change:
```
nullptr, /* tp_print */
```
to
```
0  /* tp_vectorcall_offset */
```
for Python 3.8 compatibility.

Reply:
Yeah, I don't think it's appropriate to change everything to zero. Let's just do the localized change. However, I won't block this PR on Python 3.8 CI existing.